superbench/benchmarks/micro_benchmarks/dist_inference_cpp/dist_inference.cu

```diff
@@ -328,13 +436,13 @@ void TestModel(int64_t m, int64_t n, int64_t k, float alpha, float beta, int32_t
             // cublasLt is not well supported by ROCm hipify tools, explicitly define ROCm logic instead.
 #if defined(__HIP_PLATFORM_AMD__)
             CHECK_CUBLASLT_ERROR(hipblasLtMatmul(handle, matmul1, &alpha, db, matB, da, matA, &beta, dc, matC, dd, matD,
-                                                 &heuristicResult1[0].algo, d_workspace, workspace_size, stream));
-            CHECK_CUBLASLT_ERROR(hipblasLtMatmul(handle, matmul1, &alpha, de, matE, dd, matD, &beta, df, matF, dg, matG,
```

Comment:
is this matmul1 to matmul2 a bug fix?

Reply:
I have the same question.

Reply:
It's a bug fix, but this bug will not affect previous results. Matmul1 and matmul2 happened to be exactly the same thing here.